src/Standards/Squiz/Sniffs/Scope/StaticThisUsageSniff.php

```diff
@@ -69,10 +69,11 @@ public function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScope)
         $end  = $tokens[$stackPtr]['scope_closer'];
 
         do {
-            $next = $phpcsFile->findNext([T_VARIABLE, T_CLOSURE, T_ANON_CLASS], ($next + 1), $end);
+            $next = $phpcsFile->findNext([T_VARIABLE, T_CLOSURE, T_FN, T_ANON_CLASS], ($next + 1), $end);
             if ($next === false) {
                 continue;
             } else if ($tokens[$next]['code'] === T_CLOSURE
+                || $tokens[$next]['code'] === T_FN
                 || $tokens[$next]['code'] === T_ANON_CLASS
             ) {
                 $next = $tokens[$next]['scope_closer'];
```

Comment:
T_FN tokens don't always have a scope closer, unlike closures and anon classes. This is because the T_FN token  is a real token that exists, and then scope information is added later if the syntax looks valid. But T_CLOSURE and T_ANON_CLASS are custom tokens that only get applied by PHPCS if the syntax looks good. Otherwise they are normal T_FUNCTION and T_CLASS tokens that don't have scope information.